frontend/amundsen_application/static/js/components/ResourceListItem/FeatureListItem/index.tsx

```diff
@@ -0,0 +1,84 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import { getSourceDisplayName, getSourceIconClass } from 'config/config-utils';
+import { logClick } from 'utils/analytics';
+
+import BadgeList from 'features/BadgeList';
+import { ResourceType, FeatureResource } from 'interfaces';
+
+import { LoggingParams } from '../types';
+
+export interface FeatureListItemProps {
+  feature: FeatureResource;
+  logging: LoggingParams;
+}
+
+class FeatureListItem extends React.Component<FeatureListItemProps, {}> {
+  getLink = () => {
+    const { feature, logging } = this.props;
+
+    return `/feature/${feature.key}?index=${logging.index}&source=${logging.source}`;
+  };
+
+  generateResourceIconClass = (
+    featureId: string,
+    featureType: ResourceType
+  ): string =>
+    `icon resource-icon ${getSourceIconClass(featureId, featureType)}`;
+
+  render() {
+    const { feature, logging } = this.props;
+    const source =
+      feature.availability.length > 0 ? feature.availability[0] : '';
+    return (
+      <li className="list-group-item clickable">
+        <Link
+          className="resource-list-item table-list-item"
+          to={this.getLink()}
+          onClick={(e) =>
+            logClick(e, {
```

Comment:
We are trying to move out of using logClick by triggering UI actions (plus the logClick for now).
See the example of https://github.com/amundsen-io/amundsenfrontendlibrary/blob/b5b33bba886a295df2f870677341ec4c806cd6b1/amundsen_application/static/js/ducks/bookmark/reducer.ts#L29

You might need to push down to this component the 'dispatch' method of redux-react and create an action on the UI reducer that doesn't change state.